### PR TITLE
Niranjana/email application

### PR DESCRIPTION
**Pull request by niranjana2222**
[//]: # "Feel free to customize this template and the emojis to your project's vibes"

### Description
[//]: # "Required - Describe what's new in this PR in a few lines. A description and bullet points for specifics will suffice."
- Added submitApplication so users get a confirmation email after submitting an adoption application.
- Replaced Gmail SMTP with Brevo in email.ts

## How to review
[//]: # 'Required - Describe the order in which to review files and what to expect when testing locally. Is there anything specifically you want feedback on? Should this be reviewed commit by commit, or all at once? What are some user flows to test? What are some edge cases to look out for?'
- Updated env with credentials for brevo
- Added submitApplication function on query.ts
- Updated hooks on app-process.ts and email.ts

CC: @InayaY3

### CODE REVIEW

**Review by jinkang-0 — COMMENTED** [review]
Looks great Niranjana! I just have some comments for cleaning up

File: actions/queries/query.ts [hunk 1]
```diff
@@ -81,3 +82,28 @@ export async function upsertApplication(
...
+  if (!process.env.BREVO_SMTP_USER || !process.env.BREVO_SMTP_KEY) {
+    throw new Error(
+      'Missing email configuration: BREVO_SMTP_USER and BREVO_SMTP_KEY must be set in environment variables.',
+    );
+  }
+
+  const text = `Hi! Thank you for submitting your adoption application (ID: ${app.app_uuid}). We'll review it and get back to you with a match soon.
+
+Best,
+The Adopt an Inmate Team`;
```
Comment (jinkang-0) [inline actions/queries/query.ts]: Oh I was thinking more of the literal `\n` characters. This might work too. Have you tried sending an email? The SMTP server should be setup, so calling this function should send an email.

Microscopic nit: could you also add two `\n`'s after "Hi!"? Just to match the issue. Not an important thing, I'd imagine we will change up the contents of the email in the future.

Comment (jinkang-0) [reply actions/queries/query.ts]: On the topic of reviewing/testing, could you review the "How to Review" section of the PR description? It's not apparent to me how to review from the steps there. More exposition would be appreciated.
